main_test.go

```
@@ -163,14 +164,129 @@ func TestPackageTemplateNoRenameDefaults(t *testing.T) {
+// runDoPackage builds real packages for a single arch in a temp dir and returns
+// the release dir holding them.
+func runDoPackage(t *testing.T, app, pkgOverride string) string {
+	t.Helper()
+	const rel = "RELEASE.2025-03-12T00-00-00Z"
+
+	dir := t.TempDir()
+	t.Chdir(dir)
+
+	relDir := defaultPkgName(app) + "-release"
+	archDir := filepath.Join(dir, relDir, "linux-amd64")
+	if err := os.MkdirAll(archDir, 0o755); err != nil {
```

golangci-lint (2.12.2)

[medium] G301: Expect directory permissions to be 0750 or less

(gosec)